rectools/models/nn/bert4rec.py

```diff
@@ -160,57 +159,97 @@ def _collate_fn_recommend(self, batch: List[Tuple[List[int], List[float]]]) -> D
 class BERT4RecModelConfig(TransformerModelConfig):
     """BERT4RecModel config."""
 
-    data_preparator_type: SessionEncoderDataPreparatorType = BERT4RecDataPreparator
+    data_preparator_type: TransformerDataPreparatorType = BERT4RecDataPreparator
     use_key_padding_mask: bool = True
     mask_prob: float = 0.15
 
 
 class BERT4RecModel(TransformerModelBase[BERT4RecModelConfig]):
     """
-    BERT4Rec model.
+    BERT4Rec model: transformer-based sequential model with bidirectional attention mechanism and
+    "MLM" (masked item in user sequence) training objective.
+    Our implementation covers multiple loss functions and a variable number of negatives for them.
+    See transformers tutorial: https://rectools.readthedocs.io/en/stable/examples/tutorials/transformers_tutorial.html
```

Comment:
I think it's better to remove the word `See` from each row, you sound aggressive now😂

Also there are `Notes` and `References` sections in the [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html#notes) format, you can use them

Reply:
fixed